sql/mob_pets.sql

```diff
@@ -1135,18 +1135,21 @@ INSERT INTO `mob_pets` VALUES (16928827, 1, 15); -- Tonberry_s_Elemental
 INSERT INTO `mob_pets` VALUES (16928988, 1, 15); -- Yagudo_s_Elemental
 INSERT INTO `mob_pets` VALUES (16929007, 1, 15); -- Yagudo_s_Elemental
 INSERT INTO `mob_pets` VALUES (16929015, 1, 14); -- Orc_s_Wyvern
-INSERT INTO `mob_pets` VALUES (16929055, 1, 14); -- Aern_s_Wynav
-INSERT INTO `mob_pets` VALUES (16929058, 1, 9); -- Aern_s_Euvhi
-INSERT INTO `mob_pets` VALUES (16929066, 1, 15); -- Aern_s_Elemental
--- INSERT INTO `mob_pets` VALUES (16929068, 1, 15); -- Aern_s_Avatar
-INSERT INTO `mob_pets` VALUES (16929069, 1, 14); -- Aern_s_Wynav
-INSERT INTO `mob_pets` VALUES (16929073, 1, 9); -- Aern_s_Euvhi
-INSERT INTO `mob_pets` VALUES (16929079, 1, 15); -- Aern_s_Elemental
--- INSERT INTO `mob_pets` VALUES (16929081, 1, 9); -- Aern_s_Avatar
--- INSERT INTO `mob_pets` VALUES (16929099, 1, 9); -- Aern_s_Wynav
--- INSERT INTO `mob_pets` VALUES (16929102, 1, 9); -- Aern_s_Wynav
--- INSERT INTO `mob_pets` VALUES (16929114, 1, 9); -- Aern_s_Elemental
--- INSERT INTO `mob_pets` VALUES (16929115, 1, 9); -- Aern_s_Avatar
+
+-- ------------------------------------------------------------
+-- Temenos Aerns (Zone 37)
+-- ------------------------------------------------------------
+
+INSERT INTO `mob_pets` VALUES (16929055, 1, 14); -- Temenos Aern DRG
+INSERT INTO `mob_pets` VALUES (16929058, 1, 9); -- Temenos Aern BST
```

Comment:
my records show this 58 is a rdm, and 60 is the bst, after 2 runs. 
https://github.com/project-topaz/topaz/issues/513#issuecomment-652149187

Reply:
Hm.  I see that your list in issue 513 says that 16929058 is a RDM, but

1. The NPClogger data attached to 513 shows that 16929058 has model 00005905, which is the axe model used by WAR/RNG/BST; and
2. The Aern's Euvhi with id 16929059 immediately follows this Aern, and can't belong to any other.  Pets always come after their master in the extracts (they don't always immediately follow, but they always come after; never before)

Similarly, 16929060 has model 00005A05, in both Nyu's cap and yours, which is the sword model used by RDM/PLD/DRK.  And no pet follows it in the data.

So I'm inclined to believe the jobs currently in this PR are correct.

Reply:
fair enough, data doesnt lie. thanks for taking a second look!